changelogs/current.yaml

```diff
@@ -229,6 +229,9 @@ new_features:
 - area: listener
   change: |
     added dynamic listener filter configuration for listener filters. :ref:`dynamic listener filter re-configuration<envoy_v3_api_field_config.listener.v3.ListenerFilter.config_discovery>`. This dynamic listener filter configuration is only supported in TCP listeners.
+- area: redis_proxy
+  change: |
+    added support for multiple passwords to the redis proxy.
```

Comment:
Please ref link to the new field, and per the other comment make sure you document the deprecated field.

Reply:
Done